sbt-cross/src/main/scala/sbtcross/CrossPlugin.scala

```diff
@@ -0,0 +1,24 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object CrossPlugin extends AutoPlugin {
+  override def requires: Plugins = plugins.JvmPlugin
+
```

Comment:
Hehe once again you are right! I updated the test to remove native plugins and the globalSettings were not triggered as expected.